apps/api/v2/src/modules/oauth-clients/controllers/oauth-clients/oauth-clients.controller.ts

```diff
@@ -109,6 +116,21 @@ export class OAuthClientsController {
     return { status: SUCCESS_STATUS, data: client };
   }
 
+  @Get("/managed-users/:clientId")
+  @HttpCode(HttpStatus.OK)
+  @Roles([MembershipRole.ADMIN, MembershipRole.OWNER, MembershipRole.MEMBER])
+  @DocsOperation({ description: AUTH_DOCUMENTATION })
+  async getOAuthClientManagedUsersById(
+    @Param("clientId") clientId: string
+  ): Promise<GetOAuthClientManagedUsersResponseDto> {
+    const existingManagedUsers = await this.userRepository.findManagedUsersByOAuthClientId(clientId, 0, 50); // second argument is for offset while third is for limit
+
+    if (!existingManagedUsers) {
```

Comment:
We don't need `if (!existingManagedUsers)` check because it can't be nullish. Here is its return type. If it's empty array its fine we just return empty array as data.

<img width="586" alt="Screenshot 2024-05-07 at 15 09 40" src="https://github.com/calcom/cal.com/assets/42170848/6298795d-bbd5-46f2-9360-6cc2e00f16f8">